build-to-do-list-aepp-2.md

```diff
@@ -0,0 +1,261 @@
+# TUTORIAL: How to build a To-do list Æpp - Part 2
+## Tutorial Overview
+This tutorial will teach you how to develop the communication process between the client side and the *ToDoManager* smart contract. We will setup the basic infrastructure in order to interact with the smart contract from the frontend.
+
+## Prerequisites
+- Before we go any further, please make sure you have followed the [first part](https://github.com/aeternity/tutorials/blob/master/build-to-do-list-aepp-1.md) of *How to build a To-do list Æpp*.
+- Installed **yarn** package manager ([installation](https://yarnpkg.com/lang/en/docs/install))
+
+## Plan
+I have published the simple frontend project in Github. We will start with cloning the github repository and I will explain the most important segments of the *To-do list Æpp*.
+
+## To-do list Æpp content
+The tutorial is created to showcase the aeternity SDK implementation for both Base/Wallet Aepp and simple To-do list Æpp "depending" on a base (Wallet/Identity) Aepp.
+The To-do list Æpp consists of two parts:
+- Wallet/Identity Base Aepp;
+- To-do list Æpp dependent on a wallet/identity aepp;
+
+## Getting started 
+Clone the Github repository and install the required dependencies with the following sequence of commands:
+```
+git clone https://github.com/VladislavIvanov/to-do-list-aepp.git
+cd to-do-list-aepp/identity
+yarn install
+```
+
+The Base/Wallet Aepp is started via:
+```
+yarn run start:dev
+```
+
+The Aepp runs at http://localhost:9000.
+
+Repeat the above steps for the ```aepp-origin```.
+```
+cd to-do-list-aepp/aepp-origin
+yarn install
+```
+
+The To-do list Æpp is started via:
+```
+yarn run start:dev
+```
+
+It runs at http://localhost:9001.
+
+## Wallet/Identity Base Aepp
+The Wallet/Identity Aepp that expects our Aepp to be loaded into an iFrame contained into this base aepp.
```

Comment:
can this be done with the base-aepp instead, without adjusting the code? then reference here that you are using this sinppet for simplicity.